src/conversion/conversion.js

```diff
@@ -102,6 +114,347 @@ export default class Conversion {
 		};
 	}
 
+	/**
+	 * Sets up converters between the model and the view which convert a model element to a view element (and vice versa).
+	 * For example, model `<paragraph>Foo</paragraph>` is `<p>Foo</p>` in the view.
+	 *
+	 * `definition.model` is a `String` with a model element name to converter from/to.
```

Comment:
I would put it togheter with the parameter desctription. When it is here, I ask myself why you tell me what `model` is, but not what `view` is.

Reply:
It's here because in the parameter definition specifies that input parameter is `ConverterDefinition`. I am not sure if it will be readable if such a long description would be put there.

Reply:
I know, but users will not know it. You can write that all other parameters are defined in `ConverterDefinition`, or move it after samples close to the parameter definition.

Reply:
I will move it lower.